src/api/OpenWith.js

```diff
@@ -34,25 +28,13 @@ class OpenWith extends Base {
      * @param {string} fileId - Box file ID
      * @param {Function} successCallback - Success callback
      * @param {Function} errorCallback - Error callback
-     * @param {string} locale - locale to receive translated strings from the API
      * @return {void}
      */
-    getOpenWithIntegrations(
-        fileId: string,
-        successCallback: Function,
-        errorCallback: ElementsErrorCallback,
-        locale: ?string = DEFAULT_LOCALE,
-    ) {
+    getOpenWithIntegrations(fileId: string, successCallback: Function, errorCallback: ElementsErrorCallback) {
         this.errorCode = ERROR_CODE_FETCH_INTEGRATIONS;
-        const params = {
-            headers: {
-                [HEADER_ACCEPT_LANGUAGE]: locale,
-            },
-        };
 
         this.get({
             id: fileId,
-            requestData: params,
```

Comment:
@jeremypress, can you take a look at this change?

Reply:
@JustinHoldstock where is the default locale being handled now?

Reply:
All XHR requests, as long as `language` is being passed into the `APIFactory` instance, that owns this. Right now, ContentPreview, ContentOpenWith, and ContentSidebar API calls all receive `accept-language` headers with that value